Movies Time/ViewController/MoviesListViewController.swift

```diff
@@ -97,9 +97,9 @@ extension MoviesListViewController {
 
 extension MoviesListViewController {
     override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-        let vc = MoviesDetailsViewController()
-        vc.movie = moviesList?.movies[indexPath.row]
-        self.present(vc, animated: true, completion: nil)
+        let nextView = MoviesDetailsViewController()
```

Comment:
nextView is not descriptive in the context here, it could be something like  `movieDetailsViewController`